app/controllers/lightweight_activities_controller.rb

```diff
@@ -85,6 +85,12 @@ def single_page
     @labbook_is_under_interactive = true
   end
 
+  def print_blank
+    authorize! :read, @activity
+    @run = Run.new() #Temporary run for the answer finder. Don't persist.
```

Comment:
I'm not sure, but I believe this run will get persisted by the AnswerFinder. The AnswerFinder calls `default_answer` for the Answer model associated with each question in the activity. `default_answer` then calls create on the Answer model. It refers to the run when it does this, so my guess is that the run will be persisted.   It starts here: